x-pack/plugins/maps/public/routing/bootstrap/get_initial_layers.js

```diff
@@ -41,3 +44,33 @@ export function getInitialLayers(layerListJSON, initialLayers = []) {
 
   return initialLayers;
 }
+
+export function getInitialLayersFromUrlParam() {
```

Comment:
++ Moving this function here. Should we create a type definition for this function like we did with `getInitialLayers`?

Reply:
Since the function is only used in a single place and that place is not TS, a type definition is not needed. These will be getting converted to TS shortly anyways